src/widgets/modalChildren/PatientEditModal.js

```diff
@@ -89,22 +133,44 @@ PatientEditModalComponent.propTypes = {
   onUpdateForm: PropTypes.func.isRequired,
 };
 
+const mergeProps = (stateProps, dispatchProps, ownProps) => {
+  const { completedForm, nameNote, nameNoteIsValid, surveyForm } = stateProps;
```

Comment:
There's no `nameNote` is `stateProps`. Maybe best to just spread em (have been confused in the past when returning things from a `stateToProps` to not receive it in the component because a `mergeProps` wasn't returning it.

```
const {completedForm} = stateProps;
...

return {
	...ownProps,
	...stateProps,
}
```